src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -43,15 +42,15 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
     private plot:AbstractChart;
 
     @observable mouseInPlot:boolean = false;
-    
+
     constructor(props: IChartContainerProps) {
         super(props);
         let fileName = props.clinicalAttribute.displayName.replace(/[ \t]/g,'_')
```

Comment:
fileName should be computed because otherwise it won't always reflect prop.  it's possible that this property is static, but in general we want to always make sure that our archictecture is reactive and doesn't have cached state.